### ISSUE TITLE
Feature: Goals Search

### ISSUE THREAD

**Issue by yg** [open-sauced/open-sauced#671]
### Is your feature request related to a problem? Please describe.

Searching goals in the list

### Describe the solution you'd like

We can use filter to search goals, but paginated goals will not get filtered

**Comment by bdougie** [comment]
My thought instead would be limiting the number of goals you can store. If someone is tracking more the 10 repos and issues it is likely they are not getting value out of the app or likely not contributing to anything due to decision fatigue. 

Search is not something that is a priority at the moment, but I wouldn't mind considering it with the above context.

**Comment by filiptronicek** [comment]
This is implemented in #776, right? @yg

**Comment by yg** [comment]
Yep @filiptronicek! It is completed, we will close this once merged to master!
